src/App.tsx

```diff
@@ -1,14 +1,60 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import './App.scss';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { getTodos } from './api';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { FilterStatus } from './helper';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoadedTodos, setIsLoadedTodos] = useState(false);
+  const [isTodoInfoRequested, setIsTodoInfoRequested] = useState(false);
+  const [filterCondition, setFilterCondition] = useState(FilterStatus.ALL);
+  const [query, setQuery] = useState('');
+  const [userId, setUserId] = useState<number | null>(null);
```

Comment:
Do you need this useState ? You get a user from selected todo in the moment when a modal is open